src/api.js

```diff
@@ -23,6 +25,15 @@ module.exports = function(ctx) {
     getSelectedIds: function () {
       return ctx.store.getSelectedIds();
     },
+    setSelected: function(idsToSelect) {
```

Comment:
This is duplicate functionality to `Draw.changeMode()` that automatically changes the mode for you. Once we can do this without changing the mode I think its great, but until then lets just use what we have.